src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -129,6 +145,17 @@ private StackItem SetMaxTransactionsPerBlock(ApplicationEngine engine, Array arg
             return true;
         }
 
+        [ContractMethod(0_03000000, ContractParameterType.Boolean, CallFlags.AllowModifyStates, ParameterTypes = new[] { ContractParameterType.Integer }, ParameterNames = new[] { "value" })]
+        private StackItem SetMaxBlockSystemFee(ApplicationEngine engine, Array args)
+        {
+            if (!CheckCommittees(engine)) return false;
+            long value = (long)args[0].GetBigInteger();
+            if (value <= 0) return false;
```

Comment:
```suggestion
            if (value <= 4007600) return false;
```
The `SetMaxBlockSystemFee` transaction's `SystemFee` is 4007600, the transfer native nep5 asset transaction's `SystemFee` is 9007990.
So the value of SystemFee is set to less than 4007600, that means it can never be modified back, and the blockchain is unusable.
Maybe we should control it here.

Reply:
Maybe we should take care about it and increase it a little